tests/fixtures/sample_lint_data.py

```
@@ -0,0 +1,439 @@
+            "linter": "eslint"
+        },
+        {
+            "file_path": "utils.js",
+            "line": 12,
+            "column": 1,
+            "rule_id": "no-console",
+            "message": "Unexpected console statement",
+            "severity": "WARNING",
+            "linter": "eslint"
+        }
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)